lablueprint/friends-of-the-children
# Multiple calendar functionality

Functionality:
- Allows multiple google calendars to show up on the calendar. 
- Admin profiles should see all calendars (FOTC, MS, AV), caregiver/mentor roles should only see the calendar for the area they're assigned to.
- Admin users should see an event form allowing them to add events and be able to drag/drop events, non-admin should not be able to do that
- Events should be colour-coded to their calendar

Testing:
- check that diff roles can see the appropriate calendars
- check that creating an event works and it shows up (and stays after refresh and also shows up in the actual gcal)
- check that events are the appropriate colour

TODO:
- right now calendars are hardcoded, needs to be updated eventually when passed to NPO
- bug where non-blue events don't show up in the right colour until refresh (went around the problem with force refresh but didn't actually fix)
